src/AppInstallerCLICore/Workflows/ManifestComparator.cpp

```diff
@@ -813,7 +837,7 @@ namespace AppInstaller::CLI::Workflow
             auto inapplicability = filter->IsApplicable(installer);
             if (inapplicability != InapplicabilityFlags::None)
             {
-                AICLI_LOG(CLI, Info, << "Installer " << installer << " not applicable: " << filter->ExplainInapplicable(installer));
+                AICLI_LOG(CLI, Verbose, << "Installer " << installer << " not applicable: " << filter->ExplainInapplicable(installer));

```

Comment:
I would prefer this to remain as info. When users are posting their logs, it can be an important debugging tool to identify why a certain installer is selected. Since the default is to not create verbose logs, having this at the info level makes it so we don't need to ask for verbose logs

Reply:
I changed it so that they were all at the same level.  I don't have a problem changing the entire selection process to be `Info`, but I do want it all, not just the filter pass.

Reply:
Hmm. I'll defer to the team on this then. Having it all the same makes the most sense, but I don't think it all needs to be in info. Verbose should be good enough, it's just a matter of asking for logs if we need them

Reply:
Usually by the time I see things, someone has run with / been asked to run with verbose.  But I don't know how common it is that an issue is solved with nothing more than inspecting the installer selection.

Most of the time this will generate ~5 lines in the log.  But there are some cases with 10s or over 100 installers that are a bit much.  Trying to log "the important" choices is difficult.